.up/examples/testhooks/delete-release.sh

```diff
@@ -5,6 +5,5 @@ set -aeuo pipefail
 # Note(turkenh): This is a workaround for the infamous dependency problem during deletion.
 # Note(ytsarev): In addition to helm Release deletion we also need to pause
 # XService reconciler to prevent it from recreating the Release.
-${KUBECTL} annotate xservices.aws.platformref.upbound.io --all crossplane.io/paused="true"
+${KUBECTL} annotate xservices.aws.platformref.upbound.io --all crossplane.io/paused="true" --overwrite=true
```

Comment:
why do we need `--overwrite=true` ?

Reply:
when running in MCP Controlplane this Hack is only working with --override